cli/src/main.rs

```diff
@@ -106,12 +109,103 @@ fn run() -> Result<(), CliError> {
                             Arg::with_name("url")
                                 .short("U")
                                 .takes_value(true)
+                                .default_value(DEFAULT_SPLINTER_NODE_URL)
                                 .help("URL of node"),
                         ),
                 ),
         );
     }
 
+    #[cfg(feature = "node-registry")]
+    {
+        use action::nodes;
+        use clap::{Arg, SubCommand};
+
+        app = app.subcommand(
+            SubCommand::with_name("nodes")
+                .about("Interact with a node registry")
+                .arg(
+                    Arg::with_name(nodes::URL_ARG)
+                        .long("url")
+                        .short("U")
+                        .global(true)
+                        .takes_value(true)
+                        .default_value(DEFAULT_SPLINTER_NODE_URL)
+                        .help("Splinter node's endpoint"),
+                )
+                .subcommand(
+                    SubCommand::with_name("list")
```

Comment:
List should show a table-based view by default and support csv output.

Reply:
How would arbitrary metadata be represented in a table or csv?